### PR TITLE
Ecpdocker 72

### PR DESCRIPTION
**Pull request by xorno**

### CODE REVIEW

**Comment by urvashisingh** [comment]
Same for this one. @xorno the branch is showing too many changes from previous commits that are already merged to master. Can you create a new branch from master and commit your latest change for deprecating the procedure to it and create a PR for it. Then the diffs will be only for your latest change.

File: help/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+overview: |
+  Docker is designed to deliver your applications faster. With Docker, you can separate your applications from your infrastructure and treat your infrastructure like a managed application. Docker helps you ship code faster, test faster, deploy faster, and shorten the cycle between writing code and deploying it.
```
Comment (imago-storm) [inline help/metadata.yaml]: I have moved overview into overview.md (it is too big for yaml file)

File: build/pages/help.xml [hunk 2]
```diff
@@ -0,0 +1,891 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```
Comment (imago-storm) [inline build/pages/help.xml]: We don't commit anything from build folder.

**Comment by urvashisingh** [comment]
These changes have been merged through another PR so closing this.

**Pull request closed without merging**